File: Prime.java
import java.util.*;
class Prime{
public static void main(String args[]){
Scanner sc = new Scanner(System.in);
int n;
System.out.println("enter number:");
n = sc.nextInt();
boolean flag=true;
if(n==2){
	System.out.println("it is prime");
			
}
for(int i=2;i<n;i++)
{
if(n% i==0)
{
flag=false;
break;
}
}
if(flag)
{
System.out.println("it is prime");
}
else
{
System.out.println("it is not prime!");
}
}
}

File: prime.cpp
#include<iostream>
using namespace std;
int main()
{
	int n;int flag=1;
	cout<<"enter number:";
	cin>>n;
	
	if(n<2)
	{
		cout<<"it is neither prime nor composite";
	}
	if(n==2)
	{
		cout<<"it is prime";
	}
	
	for(int i=2;i<n;i++)
	{
		if(n% i==0)
		{
			flag=0;
			break;
		}
 }
	
	if(flag)
	{
		cout<<"it is prime";
	}
	else
	{
	cout<<"it is not prime!";
	}
	
	return 0;
}
